lib/src/gotrue_admin_api.dart

```diff
@@ -0,0 +1,109 @@
+import 'package:gotrue/gotrue.dart';
+import 'package:gotrue/src/fetch.dart';
+import 'package:gotrue/src/types/fetch_options.dart';
+import 'package:http/http.dart';
+
+class GoTrueAdminApi {
+  final String _url;
+  final Map<String, String> _headers;
+
+  final Client? _httpClient;
+  late final GotrueFetch _fetch = GotrueFetch(_httpClient);
+
+  GoTrueAdminApi(
+    this._url, {
+    Map<String, String>? headers,
+    Client? httpClient,
+  })  : _headers = headers ?? {},
+        _httpClient = httpClient;
+
+  /// Removes a logged-in session.
+  Future<void> signOut(String jwt) async {
+    final headers = {..._headers};
+    headers['Authorization'] = 'Bearer $jwt';
```

Comment:
Can't we simply pass the headers to the `GotrueRequestOptions`?

Reply:
Maybe I'm missing something, but don't I created the variable `headers` so that we can override the `Authorization` header in line 23, so I don't think we can pass the headers directly to `GotrueRequestOptions` here.

Reply:
Oh I've written the wrong. I meant we can pass the `jwt` to `GoTrueRequestOptions`, because it overrides the header [here](https://github.com/supabase-community/gotrue-dart/pull/90/files#diff-fb71a9f29effc766e3d7775f310b2b560d7abe17048f0280e5ce605517d983f1R46) by itself.

Reply:
@Vinzent03 Ah, amazing eye! I totally missed that. Pushing a fix now!